articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+
+3. In Test Failover, select the target Azure network.
+
+4. Select OK to begin the failover.
+
+    ![Azure replication failover](images/azs-browser-recovery-failover.png)
+
+5. Track progress by selecting the VM to open its properties. Or, select the **Test Failover Job** in vault name > Settings > Jobs >Site Recovery jobs.
+
+    ![Azure replication test failover](images/azs-browser-recovery-testfailover.png)
+
+6. After the failover finishes, the replica Azure VM appears in the Azure portal > Virtual Machines. Check that the VM is the appropriate size, connected to the right network, and running.
+
+7. You should now be able to connect to the replicated VM in Azure.
+
+    1. To connect to the VM via RDP you will need to assign a public IP to the VM.
+
+    2. Navigate to the VMs **Network interface**.
+
+    3. Under settings select **IP configurations**, then select the config.
+
+    4. Enable **Public IP address**.
+
+    5. Configure **IP address** and create a new IP and select **OK**.
+
+    6. Select save to create the public IP for the VM.
+
+8. To delete Azure VMs created during the test failover, select **Cleanup test failover** on the VM. In Notes, save any observations associated with the test failover.
+
+## Fail Over and Fail Back
+
+After you have finished setting up replication, and ran a test failover to make sure everything's working as expected, you can fail machines over to Public Azure as needed.
+
+Run a failover as follows:
+
+1. In Settings navigate to **Replicated Items**, select the machine then **Failover**.
+
+2. Select the recovery point that you wish to use for the failover.
+
+3. Select **OK** to begin the failover. You can follow the failover progress on the Jobs page.
+
+4. After the failover finishes, the replica Azure VM appears in the Azure portal in the Virtual Machines tab. If you prepared to connect after failover, check that the VM is the appropriate size, connected to the right network, and running.
+
+    * Remember to add a public IP to the VM if it needs one, without a public IP you will be unable to RDP to the VM.
+
+5. After verifying the VM has correctly failed over, select Commit to finish the failover. This deletes all available recovery points for the VM.
+
+> [!CAUTION]
+> Don't cancel a failover in progress: Before failover is started, VM replication is stopped. If you cancel a failover in progress, failover stops, but the VM won't replicate again.
+
+### Fail back to Azure Stack
+
+When you primary site is up and running again, you can fail back from Azure to Azure Stack. To do this, you need to download the Azure VM VHD, and upload it to Azure Stack.
+
+1. Shut down the Azure VM, so that the VHD can be downloaded.
+
+2. Navigate to your disks on the VM
+
+3. Select a **Disk**
+
+4. Navigate to **Disk Export**
+
+5. Select **Export** > **Download VHD**
+
+6. Upload the VHD to Azure Stack with these [*steps*](azs-how-manage-disks.md).
```

Comment:
Upload the VHD to Azure Stack by following the steps in [*title*](link).

(If this is supposed to link to the other file in this pull request, I think you have the file name wrong here)

Reply:
done